menu.js

```diff
@@ -20,3 +21,195 @@ navMenu.addEventListener('click', () => {
+  const bgdivUlLi2 = document.createElement('li');
+  const bgdivUlLi3 = document.createElement('li');
+  bgdivUl.append(bgdivUlLi1, bgdivUlLi2, bgdivUlLi3);
+  bgdivUlLi1.className = 'work1';
+  const subwork1index = popupObject[i].subwork[0];
+  const subwork2index = popupObject[i].subwork[1];
+  const subwork3index = popupObject[i].subwork[2];
+  bgdivUlLi1.textContent = subwork1index;
+  bgdivUlLi2.textContent = subwork2index;
+  bgdivUlLi3.textContent = subwork3index;
+  const bgdivP = document.createElement('p');
+  bgdivP.textContent = popupObject[i].description;
+  bgdiv.appendChild(bgdivP);
+  bgdivP.className = 'about';
+  const languagesUl = document.createElement('ul');
+  languagesUl.className = 'languages';
+  bgdiv.appendChild(languagesUl);
+  const languagesUlLi1 = document.createElement('li');
+  const technologiesindex1 = popupObject[i].technologies[0];
+  languagesUlLi1.textContent = technologiesindex1;
+  const languagesUlLi2 = document.createElement('li');
+  const technologiesindex2 = popupObject[i].technologies[1];
+  languagesUlLi2.textContent = technologiesindex2;
+  const languagesUlLi3 = document.createElement('li');
+  const technologiesindex3 = popupObject[i].technologies[2];
+  languagesUlLi3.textContent = technologiesindex3;
+
+  const languagesUlLi4 = document.createElement('li');
+  const languagesUlLi5index1 = popupObject[i].technologies[3];
+  languagesUlLi4.textContent = languagesUlLi5index1;
+  const languagesUlLi5 = document.createElement('li');
+  const languagesUlLi5index2 = popupObject[i].technologies[4];
+  languagesUlLi5.textContent = languagesUlLi5index2;
+  const languagesUlLi6 = document.createElement('li');
+  const languagesUlLi6index3 = popupObject[i].technologies[5];
+  languagesUlLi6.textContent = languagesUlLi6index3;
+  languagesUl.append(languagesUlLi1, languagesUlLi2, languagesUlLi3);
+  languagesUl.append(languagesUlLi4, languagesUlLi5, languagesUlLi6);
+  languagesUlLi4.style.display = 'none';
+  languagesUlLi5.style.display = 'none';
+  languagesUlLi6.style.display = 'none';
+  const btnDiv = document.createElement('div');
+  bgdiv.appendChild(btnDiv);
+  const btn = document.createElement('button');
+  btn.className = 'clickme';
+  btnDiv.appendChild(btn);
+  btn.textContent = 'See Projects';
+  btn.addEventListener('click', () => {
+    if (window.screen.width > 992) {
+      languagesUlLi4.style.display = 'inline';
+      languagesUlLi5.style.display = 'inline';
+      languagesUlLi6.style.display = 'inline';
+    }
+    const popupWindowMainCont = document.createElement('section');
+    workSectionContainer.appendChild(popupWindowMainCont);
+    popupWindowMainCont.className = 'popupWindowCont';
+    const popupwindow = document.createElement('section');
+    popupWindowMainCont.appendChild(popupwindow);
+    popupwindow.className = 'popupWindow';
+    const closeIcon = document.createElement('img');
+    closeIcon.src = 'images/xIcon.svg';
```

Comment:
Please be sure that your close button image is showing 🧐

![image](https://user-images.githubusercontent.com/15807343/157932017-63b8fb99-8ee4-4e69-8e58-6dfed19ca210.png)

Reply:
resolved 👍